api/cluster/resource/templater_test.go

```diff
@@ -4482,3 +4841,39 @@ func createPyFuncPublisherEnvVars(svc *models.Service, pyfuncPublisher config.Py
 	}
 	return envVars
 }
+
+func sortInferenceServiceSpecEnvVars(isvc kservev1beta1.InferenceServiceSpec) {
```

Comment:
This helper function is only used in this test in order to sort out all the env vars in an inference service spec, so that any method performing a comparison on 2 inference services does not throw errors caused by differences in the order of env vars.